### PR TITLE
Update IDE Addin script to use the Build Number

### PR DESCRIPTION
**Pull request by dellis1972**
@tomspilman do I need to revert the change to the addin.xml after the build or will team city just ignore it?

### CODE REVIEW

**Comment by tomspilman** [comment]
I guess you are retrying this in a new PR?  Or are you needing something from me?

**Comment by tomspilman** [comment]
> do I need to revert the change to the addin.xml after the build or will team city just ignore it?

I am pretty sure the agent will revert any changes to files before the build.

**Comment by dellis1972** [comment]
@tomspilman sorry , I forgot a few scripts :) Ok cool :)

**Comment by tomspilman** [comment]
Cool... lets see what the build bot says.

**Comment by tomspilman** [comment]
Checking the server to be sure the number updated correctly.

**Comment by tomspilman** [comment]
@dellis1972 - Can the script maybe delete existing `MonoGame_*.mpack` files before creating a new one?

**Comment by dellis1972** [comment]
you I'll update the script to delete it

**Comment by dellis1972** [comment]
@tomspilman done :)

File: IDE/MonoDevelop/default.build [hunk 1]
```diff
@@ -44,6 +49,9 @@
      <get src="https://www.nuget.org/api/v2/package/MonoGame.Framework.Content.Pipeline.Portable/3.2.99.1-Beta" dest="MonoDevelop.MonoGame/packages/MonoGame.Framework.Content.Pipeline.Portable.3.2.99.1-Beta.nupkg" />
      <get src="https://www.nuget.org/api/v2/package/Ouya.Console.Api/1.0.13.1" dest="MonoDevelop.MonoGame/packages/Ouya.Console.Api.1.0.13.1.nupkg"/>
      <if test="${os == 'MacOS'}">
+       <delete file="MonoDevelop.MonoGame_*.mpack" failonerror="false" />
```
Comment (tomspilman) [inline IDE/MonoDevelop/default.build]: This doesn't seem to have worked... the files are still on the build agent.

I suspect that the `<delete>` task has to be used with a `<fileset>` to use wildcards.

**Comment by tomspilman** [comment]
Looks like this worked... thanks @dellis1972 !

**Pull request merged**